CHANGELOG.md

```diff
@@ -30,6 +31,7 @@
 * [BUGFIX] Memberlist: fixed corrupted packets when sending compound messages with more than 255 messages or messages bigger than 64KB. #4601
 * [BUGFIX] Query Frontend: If 'LogQueriesLongerThan' is set to < 0, log all queries as described in the docs. #4633
 * [BUGFIX] Distributor: update defaultReplicationStrategy to not fail with extend-write when a single instance is unhealthy. #4636
+* [BUGFIX] Fix data race when stopping chunks memcached client. #4511
```

Comment:
This is not expected to be user-impacting either; it was detected in a test.

Reply:
I see, I will remove.